src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -98,7 +98,9 @@ def load_arguments(self, _):
     with self.argument_context('resource create') as c:
         c.argument('resource_id', options_list=['--id'], help='Resource ID.', action=None)
         c.argument('properties', options_list=['--properties', '-p'], help='a JSON-formatted string containing resource properties')
-        c.argument('is_full_object', action='store_true', help='Indicates that the properties object includes other options such as location, tags, sku, and/or plan.')
+        c.argument('is_full_object', action='store_true', help='Indicate that the properties object includes other options such as location, tags, sku, and/or plan.')
+        c.argument('latest_version', is_preview=True, action='store_true',
+                   help='Indicate that the latest api-version is used regardless of whether it is "preview" or not (omit for using the latest stable api-version)')
```

Comment:
If we use `allow_preview`, this line could be:

```suggestion
                   help='Indicate that preview versions (like 2016-01-01-preview) are included when determine the latest API version.')
```